### ISSUE TITLE
What is, and where to get, dispnorm_occ for Kitti 2015?

### ISSUE THREAD

**Issue by andrewlstewart** [meteorshowers/X-StereoLab#36]
I downloaded the [stereo 2015/flow 2015/scene flow 2015 data set (2 GB) ](http://www.cvlibs.net/datasets/kitti/eval_scene_flow.php?benchmark=stereo) but I don't see any subfolder with the name 'dispnorm_occ'.  Without it, I can't call disparity.KITTIloader2015.dataloader to actually train the StereoNet model.  Can you help me find it or explain how to generate it?

Thanks!

**Comment by Yaoxingtian** [comment]
> I downloaded the [stereo 2015/flow 2015/scene flow 2015 data set (2 GB) ](http://www.cvlibs.net/datasets/kitti/eval_scene_flow.php?benchmark=stereo) but I don't see any subfolder with the name 'dispnorm_occ'. Without it, I can't call disparity.KITTIloader2015.dataloader to actually train the StereoNet model. Can you help me find it or explain how to generate it?
> 
> Thanks!

Hi , do you solve this problem ? I meet this problem to . 
Could you kindly share info if you know how to do ? thanks much !

**Comment by andrewlstewart** [comment]
@Yaoxingtian I found this to be a somewhat broken repo.  I wasn't able to find the dispnorm_occ files.  I originally came here looking for a StereoNet implementation but couldn't get their implementation working.  I implemented the StereoNet framework here: github.com/andrewlstewart/StereoNet_PyTorch

Hope that helps

**Comment by Yaoxingtian** [comment]
yes, I thinks so , this is a half-done work. they didnot open all source code .


发送自 Outlook<http://aka.ms/weboutlook>

________________________________
发件人: Andrew Stewart ***@***.***>
发送时间: 2021年11月24日 15:43
收件人: meteorshowers/X-StereoLab ***@***.***>
抄送: Yaoxingtian ***@***.***>; Mention ***@***.***>
主题: Re: [meteorshowers/X-StereoLab] What is, and where to get, dispnorm_occ for Kitti 2015? (#36)


@Yaoxingtian<https://github.com/Yaoxingtian> I found this to be a somewhat broken repo. I wasn't able to find the dispnorm_occ files. I originally came here looking for a StereoNet implementation but couldn't get their implementation working. I implemented the StereoNet framework here: github.com/andrewlstewart/StereoNet_PyTorch

Hope that helps

―
